src/Microsoft.Azure.WebJobs.Host/JobHost.cs

```diff
@@ -123,10 +123,12 @@ public Task StartAsync(CancellationToken cancellationToken = default(Cancellatio
 
         private async Task StartAsyncCore(CancellationToken cancellationToken)
         {
-            await EnsureHostStartedAsync(cancellationToken);
```

Comment:
As you can see by the fact that below we're starting the listeners, this EnsureHostStarted method was named incorrectly